R/pin-read-write.R

```diff
@@ -64,6 +64,10 @@ pin_read <- function(board, name, version = NULL, hash = NULL, ...) {
 #'   use the default for `board`
 #' @param tags A character vector of tags for the pin; most important for
 #'   discoverability on shared boards.
+#' @param compare_hash Compare the pin contents to the last version if one
```

Comment:
I think this is an improvement, but I wondered about something even more direct like `write_if_different`? Or `write_unchanged_data`? What do you think?

Reply:
What about `force_identical_write`?

Reply:
None of these ideas really speak to me (including mine), so just go with your heart 😄